stdlib/public/core/EmbeddedRuntime.swift

```diff
@@ -149,6 +149,13 @@ func alignedAlloc(size: Int, alignment: Int) -> UnsafeMutableRawPointer? {
   return unsafe r
 }
 
+@_cdecl("swift_coroFrameAlloc")
+public func swift_coroFrameAlloc(_ size: Int, _ type: UInt) -> UnsafeMutableRawPointer? {
```

Comment:
An alternative would be to set `EmitTypeMallocForCoroFrame` to false on Embedded.

Reply:
I'll leave that policy decision to someone else.